wger/manager/views/workout.py

```diff
@@ -53,6 +57,44 @@
 logger = logging.getLogger(__name__)
 
 
+@login_required
+def export_workouts(request):
+        '''
+        Exports workouts
+        '''
+        # get all workouts
+        workouts = Workout.objects.filter(user=request.user)
+        # generate json data
+        data = serializers.serialize("json", workouts)
+        # create json response
+        response = HttpResponse(data, content_type='text/json')
+        # send response to data.json file
+        response['Content-Disposition'] = 'attachment; filename=data.json'
+        return response
+
+
+def import_workouts(request):
+    file = request.FILES['import_file']
+    data = file.read()
+    f = open('data.json', 'r')
+    import json
+    json1_data = json.loads(f.read())
+    for value in json1_data:
+        value['fields']['user'] = request.user.id
+        value['pk'] = random.randint(50, 1909)
+    for deserialized_object in serializers.deserialize("json", json.dumps(json1_data)):
```

Comment:
Ideally you should be adding/creating the imported workouts. Not really sure what's going on here. Please check the `WorkoutEditView` class and `add`/`copy_workout` methods on this file to get an idea of how you should do this.

Reply:
The script above is adding imported workouts, The workouts that were exported from  another instance